addons/portal_rating/views/assets.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="assets_frontend" inherit_id="web_editor.assets_frontend" name="Portal Rating Assets">
+        <xpath expr="//link[last()]" position="after">
+            <link rel="stylesheet" type="text/scss" href="/portal_rating/static/src/scss/website_rating.scss"/>
```

Comment:
Remaining occurence of website_rating on portal

Reply:
Done, and also re-namespaced some templates (website_rating.brol to portal_rating.brol).

I did not change class names as this would imply a lot of diff.